### PR TITLE
Consistent schema format between websocket endpoint and ksql endpoint

### PR DESCRIPTION
**Pull request by rodesai**
### Description 
Serialize the schema description entities from the API endpoint, as opposed to
the schema itself. This is consistent with the ksql endpoint representation of
schemas for stream/table description results.

Remove the SchemaMapper since it is no longer used by anything.

### Testing done 
_Describe the testing stategy. Unit and integration tests are expected for any behavior changes._

### Reviewer checklist
- [ ] Ensure docs are updated if necessary. (eg. if a user visible feature is being added or changed).
- [ ] Ensure relevant issues are linked (description should include text like "Fixes #<issue number>")

### CODE REVIEW

File: ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java [hunk 1]
```diff
@@ -48,7 +48,7 @@
   private static final ObjectMapper mapper = JsonMapper.INSTANCE.mapper;
 
   static {
-    new SchemaMapper().registerToObjectMapper(mapper);
+    mapper.registerModule(new Jdk8Module());
```
Comment (xvrl) [inline ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java]: ideally we would create a single ObjectMapper, add all the modules in one place, and then inject that into various places we need them, not sure if there is a plan to do that.

File: ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriber.java [hunk 2]
```diff
@@ -98,7 +99,7 @@ public void onComplete() {
   public void onSchema(Schema schema) {
     try {
       session.getBasicRemote().sendText(
-          mapper.writeValueAsString(schema)
+          mapper.writeValueAsString(EntityUtil.buildSourceSchemaEntity(schema))
```
Comment (xvrl) [inline ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriber.java]: where do we draw the line internally on when we use connect `Schema`s vs. where we use `SchemaInfo` schemas? Is it only for the rest APIs or do we also use `SchemaInfo` internally?

Comment (rodesai) [reply ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriber.java]: Its just for the API.

File: ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java (hunk 1 above)
Comment (rodesai) [reply ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/streaming/WebSocketSubscriberTest.java]: There actually was a recent change to use a common ObjectMapper - though its accessed as a singleton, not injected. So this line is actually unnecessary. I'll take it out.

**Pull request merged**